dz8/controller.py

```diff
@@ -0,0 +1,49 @@
+import operations
+import user_interface as ui
+
+process = True
+ui.start_message()
+i = input()
+while not i == '1' or not i == '2':
```

Comment:
```suggestion
while i != '1' or i != '2':
```
А чем вам такая запись не устраивает?

Reply:
И вообще, у вас ведь будет всегда истина, какое бы вы `i` не передали. Так что можно просто написать:
```suggestion
while True:
```